projects/packages/jetpack-mu-wpcom/src/features/launchpad/launchpad-task-definitions.php

```diff
@@ -343,13 +343,22 @@ function wpcom_launchpad_get_task_definitions() {
 			},
 			'is_complete_callback' => 'wpcom_is_task_option_completed',
 			'is_visible_callback'  => 'wpcom_is_enable_subscribers_modal_visible',
+			'get_calypso_path'     => function ( $task, $default, $data ) {
+				if ( ( new Automattic\Jetpack\Status\Host() )->is_atomic_platform() ) {
+					return 'https://' . $data['site_slug_encoded'] . '/wp-admin/admin.php?page=jetpack#/discussion';
```

Comment:
I think we should use `admin_url()` for this use case rather than building it dynamically. ([docs](https://developer.wordpress.org/reference/functions/admin_url/))
```suggestion
					return admin_url( '/admin.php?page=jetpack#/discussion' );
```

Reply:
Thanks! I fixed it. However, one of the rules you added prevents from having a full URL on the `get_calypso_path` prop. Do you think we can remove this rule?

Reply:
Hmmm... Maybe we can allow URLs that start with `admin_url()`?